src/plugins/intel_gna/src/ops/gna_convolution.hpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace op {
+
+class GNAConvolution;
+
+namespace internal {
```

Comment:
wouldn't be better to have internal stuff only in source file or eventually in separate header/source file?